### DISCUSSION TITLE
The network stability with China location

### DISCUSSION THREAD

**Discussion by TheWithCommands** [community/community#40191] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Recently, Github related services in China were slow to access or could not be opened, and the files(e.g. releases) could not be downloaded normally. They could only be accessed reluctantly through related tools, but still could not be downloaded normally.

**Comment by ossanna16** [comment]
Hi there @TheWithCommands and welcome to our community! Thank you for reporting this!

**Comment by Annran233** [comment]
现在GitHub已经完全挂了（悲）
now it can't access in China (sadly)

**Comment by ghost** [comment]
You can use a reverse proxy to speed up, no tools are required,If you can't make it, I have a repository about it
